src/main/java/robot/subsystems/hatchIntake/commands/ExtensionCommand.java

```diff
@@ -0,0 +1,42 @@
+package robot.subsystems.hatchIntake.commands;
+
+import edu.wpi.first.wpilibj.command.Command;
+import robot.Robot;
+
+public class ExtensionCommand extends Command {
```

Comment:
There is no need to write the word "Command" in a command name.
Also, a command name should include a verb and not a gerund.